gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.coordinates import SkyCoord
+
+from ...data import DataStore, ObservationTable, ObservationTableSummary
+
+from ...utils.testing import data_manager, requires_data, requires_dependency
+
+def init_summary():
```

Comment:
For this I would recommend you put a [pytest.fixture](https://pytest.org/latest/fixture.html#fixtures-as-function-arguments). Such fixtures look very weird at first, but they are very powerful and a very good way to create test objects and "inject" them into test functions.

In this case I think this should do it (I didn't test):

```python
@requires_data('gammapy-extra')
def summary():
    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
    target_pos = SkyCoord(83.633083, 22.0145, unit='deg')
    return ObservationTableSummary(data_store.obs_table, target_pos)
```

And then you use the fixture like this, i.e. inject it as an input parameter into the test function with the same name as the fixture function above:
```python
def test_str(summary):
    text = str(summary)
    assert 'Observation summary' in text
```